lib/httpServer.js

```diff
@@ -62,20 +56,23 @@ const facts = async context => {
 }
 
 const app = new Koa()
-app.use(cors())
+app.use(cors)
 app.use(body())
 if (opts.verbose) {
   app.use(log())
 }
-app.use(route.get('/ping', async context => context.body = 'pong'))
-app.use(route.all('/assert', assert))
-app.use(route.all('/select', select))
-app.use(route.all('/retract', retract))
-app.use(route.all('/retractEverythingAbout', retractEverythingAbout))
-app.use(route.all('/facts', facts))
+app.use(route.get('/ping', async context => { context.body = 'pong' }))
+app.use(route.post('/assert', assert))
+app.use(route.post('/select', select))
+app.use(route.post('/retract', retract))
+app.use(route.get('/facts', facts))
 app.use(static_('examples'))
 
 module.exports = client => {
   app.context.client = client
+  // should be app.on('listening', () => {
+  const name = process.env.LIVING_ROOM_NAME || require('os').hostname()
+  const bonjour = require('nbonjour').create()
+  bonjour.publish({name, type: 'http', protocol: 'tcp', subtypes: ['livingroom'], port: 3000})
```

Comment:
Not sure if we should export name as our instantiation function.

I also wonder if we should run `.listen()` here